### ISSUE TITLE
The imshow of celebA project

### ISSUE THREAD

**Issue by RongKaiWeskerMA** [znxlwm/pytorch-MNIST-CelebA-GAN-DCGAN#9]
In celebA project, 

 ax[i, j].imshow((test_images[k].cpu().data.numpy().transpose(1, 2, 0) + 1) / 2)

is there any reason for you to make the image (value +1)/2?

**Comment by saurabheights** [comment]
Conversion from -1, 1 to 0,1 range. This is to invert normalization transform of images done in dataloader.

@RongKaiWeskerMA
